lib/main_micro_alp.pm

```diff
@@ -32,6 +32,11 @@ sub is_dvd {
     return get_required_var('FLAVOR') =~ /dvd/i;
 }
 
+sub is_qec_test_run {
+    my $vx = get_var('QEC_TEST_FILTER');
```

Comment:
```suggestion
    return check_var('SLEM_IMAGE_TESTS', '1');
```
`QEC_TEST_FILTER` appears too generic to me

Reply:
OK; but being it dedicated to ownership, then better: `QEC_SLEM_IMAGE_TESTS`

Reply:
Also, I considered the case when param. `undefined` no filtering shall be applied

Reply:
But, another consideration on name: this new param. is not only for image tests, even being now used for that case only :slightly_smiling_face: so I'd prefer to call it `QEC_SLEM_TESTS`

Reply:
QEC runs container engine, SLEM image and BCI tests on SLE-Micro. We have already now three different test cases, so I'd argue that `QEC_SLEM_TESTS` or `QEC_SLEM_TESTS_FILTER` does not distinguish between them sufficiently.